packages/blade/src/components/Input/BaseInput/BaseInput.tsx

```diff
@@ -147,6 +152,60 @@ export type BaseInputProps = InputLabelProps &
      * Displays the character counter under the input field
      */
     maxCharacters?: number;
+    /**
+     * alignement of the text inside input field
```

Comment:
```suggestion
     * alignment of the text inside input field
```

Reply:
There's another issue above in the prop spelling but I can't comment there so posting here

```diff
- neccessityIndicator?: FormLabelProps['necessityIndicator'];
+ necessityIndicator?: FormLabelProps['necessityIndicator'];
```

Reply:
fixed in b8929d24b4e7a0d218bba7e599803658e69dc4a8